src/main/java/analyzer/exercises/needforspeed/AvoidConditionalLogic.java

```diff
@@ -0,0 +1,15 @@
+package analyzer.exercises.needforspeed;
+
+import analyzer.Comment;
+
+public class AvoidConditionalLogic extends Comment {
+    @Override
+    public String getKey() {
+        return "java.needforspeed.avoid_conditional_logic";
```

Comment:
The key defines the path in the website-copy repository where the template is located, in this case that would be `java/needforspeed/avoid_conditional_logic.md`. 

I would prefer if we store the templates in directories named after the exercise slug, so `java/need-for-speed/avoid_conditional_logic.md`, which means they key here should be `java.need-for-speed.avoid_conditional_logic`.

The same goes for the other comments as well.

Reply:
Done!